src/main/java/edu/unc/lib/boxc/migration/cdm/status/SourceFilesSummaryService.java

```diff
@@ -20,64 +28,57 @@
  */
 public class SourceFilesSummaryService extends AbstractStatusService {
     private static final Logger log = getLogger(SourceFilesSummaryService.class);
+    private static final String ID_FIELD = "id";
+    private static final String MATCHING_VALUE_FIELD = "matching_value";
+    private static final String SOURCE_FILE_FIELD = "source_file";
+    private static final String POTENTIAL_MATCHES_FIELD = "potential_matches";
 
     private SourceFileService sourceFileService;
     private boolean dryRun;
     private int previousStateFilesMapped;
+    private List<CSVRecord> previousStatePopulatedFiles;
+    private int sampleSize = 20;
 
     /**
      * Display summary about source file mapping
      * @param verbosity
      */
     public void summary(Verbosity verbosity) {
-        int previousFilesMapped = previousFilesMapped();
         int totalFilesMapped = totalFilesMapped();
-        int newFilesMapped = newFilesMapped(totalFilesMapped, previousFilesMapped);
+        int newFilesMapped = totalFilesMapped - previousStateFilesMapped;
         int totalObjects = totalFilesInProject();
+        List<CSVRecord> listFiles = listNewFiles();
 
         if (verbosity.isNormal()) {
-            showField("Previous Files Mapped", previousFilesMapped);
+            showField("Previous Files Mapped", previousStateFilesMapped);
             showField("New Files Mapped", newFilesMapped);
             showField("Total Files Mapped", totalFilesMapped);
             showField("Total Files in Project", totalObjects);
+            showFiles(listFiles);
         }
     }
 
-    /**
-     * @return number of new files mapped
-     */
-    public int newFilesMapped(int totalFilesMapped, int previousFilesMapped) {
-        return totalFilesMapped - previousFilesMapped;
-    }
-
     /**
      * @return total number of files mapped
      */
     public int totalFilesMapped() {
         return countFilesMapped(getNewMappingPath());
     }
 
-    /**
-     * @return previous number of files mapped
-     */
-    public int previousFilesMapped() {
-        return previousStateFilesMapped;
-    }
-
     /**
      * @return number of files mapped
      */
     public int countFilesMapped(Path mappingPath) {
         Set<String> indexedIds = getQueryService().getObjectIdSet();
         Set<String> mappedIds = new HashSet<>();
         try {
-            SourceFilesInfo info = SourceFileService.loadMappings(mappingPath);
-            for (SourceFilesInfo.SourceFileMapping mapping : info.getMappings()) {
-                if (mapping.getSourcePaths() != null && indexedIds.contains(mapping.getCdmId())) {
-                    mappedIds.add(mapping.getCdmId());
+            List<CSVRecord> populatedEntries = loadPopulatedEntries(mappingPath);
+            for (CSVRecord entry : populatedEntries) {
+                if (!entry.get(SOURCE_FILE_FIELD).isEmpty() && indexedIds.contains(entry.get(ID_FIELD))) {
```

Comment:
since we're starting from loadPopulatedEntries, we should not need to check `!entry.get(SOURCE_FILE_FIELD).isEmpty()` since that happens inside the other method.